test/jdk/java/awt/PopupMenu/PopupMenuLocation.java

```diff
@@ -95,6 +111,7 @@ public void mouseReleased(MouseEvent e) {
 
                 private void show(MouseEvent e) {
                     if (e.isPopupTrigger()) {
+                        System.out.println("Going to show popup "+pm+" on "+frame);
```

Comment:
There seems to be a lack of consistency with spacing throughout the test. Spacing out the "+" from the quotations and the vars makes it cleaner.